sanic/handlers.py

```diff
@@ -83,16 +83,16 @@ def response(self, request, exception):
             if response is None:
                 response = self.default(request, exception)
         except Exception:
-            self.log(format_exc())
```

Comment:
took me a second to grok this but I think part of the point of have the `self.log` function on the `ErrorHandler` class was that the class should funnel logs through it so the method could be overridden in a user subclass. I think removing the call here would be backwards incompatible with user code so instead of doing `logger.exception(...)` below (line 92) we need to do `self.log(..., 'exception')`. the problem with that is the function only takes 1 argument where as `logger.exception` takes multiple which makes using `self.log` how you want also backwards incompatible 😞 . I think changing the signature of log to `def log(message, *args, level='error')` is probably less backwards incompatible but would like to hear some other opinions on this

Reply:
I agree, we still need to use `self.log()` here rather than `logger.exception()`, however we need to be able to pass in the extra arguments.
I think changing the log signature to `log(message, *args, level='error')` is a great solution. I'd like to see that changed as part of this PR.

Reply:
How is this more backwards compatible? Anybody who previously overwrote `log` will not be able to deal with this invocation. If anything this makes it worse because it will crash while crashing vs just not using their `log`